### PR TITLE
[PowerToys]Document new Mouse Jump utility

### PR DESCRIPTION
**Pull request by jaimecbernardo**
Adds documentation for the new Mouse Jump mouse utiltity in PowerToys.

### CODE REVIEW

**Comment by jaimecbernardo** [comment]
@crutkas

**Comment by jaimecbernardo** [comment]
@mattwojo , this should only go in after 0.68 is released. Thank you!

**Review by stefansjfw — APPROVED** [review]
Lgtm

**Review by crutkas — APPROVED** [review]

File: hub/powertoys/mouse-utilities.md [hunk 1]
```diff
@@ -58,6 +59,16 @@ From the settings menu, the following options can be configured:
...
 
+## Mouse jump
+
+![Screenshot of Mouse jump.](../images/pt-mouse-jump.gif)
+
+Mouse Jump allows moving the mouse pointer long distances on a single screen or across multiple screens.
+
+| Setting | Description |
+| :--- | :--- |
+| Activation shortcut | The customizable keyboard command to activate Mouse Jump. |
```
Comment (Jay-o-Way) [inline hub/powertoys/mouse-utilities.md]: Should it be a big "J" or a "j"?

Comment (mattwojo) [reply hub/powertoys/mouse-utilities.md]: Good catch. Small j to align with how we cased previous Mouse ___ behavior.

**Comment by Jay-o-Way** [comment]
@mattwojo v0.68 was released like 10 minutes ago 🙂

**Comment by mattwojo** [comment]
@crutkas - Do you have a list of doc updates for v0.68? Ok to merge this update?

**Review by mattwojo — APPROVED** [review]

**Pull request merged**